modules/core/04-channel/keeper/packet.go

```diff
@@ -33,7 +33,7 @@ func (k Keeper) SendPacket(
 		return 0, sdkerrors.Wrap(types.ErrChannelNotFound, sourceChannel)
 	}
 
-	if channel.State == types.CLOSED {
+	if channel.State != types.OPEN {
```

Comment:
Should we add a test like [this one](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel/keeper/packet_test.go#L98-L104), but for `TRY_OPEN`?